package.json

```diff
@@ -33,21 +33,29 @@
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
+    "@fluffy-spoon/substitute": "^1.107.0",
```

Comment:
https://www.npmjs.com/package/@fluffy-spoon/substitute - selected for simplicity and good typing support. It does not support mock resets which I find as a good thing. Shared mocks that needs to be reset before every test are problematic in long terms.